### DISCUSSION TITLE
Build duration limit? "Build container is stuck in BUILDING state for 2700000ms"

### DISCUSSION THREAD

**Discussion by tomgreeEn** [vercel/vercel#5572] [General]
Is there a limit on build times? I'm building a fairly large site (1300 pages) which keeps failing after 45 minutes with the message: `Build container is stuck in BUILDING state for 2700000ms`

Is this because it's hitting a build duration limit? I can't find any reference to build duration limits in the docs. I'm on the hobby plan but happy to upgrade to Pro if this gets around this limit.

Thanks

**Comment by AlexandreUser** [comment]
I have the same issue, but i'm rendering a page with 13000 static files.
